src/Presentation/Views/ViewAccount/ViewAccount.tsx

```diff
@@ -0,0 +1,11 @@
+
+function ViewAccount() {
```

Comment:
I think the use of Login / Registration makes a lot more sense, is there any reason why we have ViewAccount and EditAccount? It seems like ViewAccount might be more for viewing other user's profiles if we add in a friends feature.

Reply:
Yeah, I was just thinking about removing the ViewAccount after I pushed. It's probably not necessary right now.